modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/admin/EventProcessorAdminServiceTestCase.java

```diff
@@ -0,0 +1,196 @@
+            Assert.assertEquals(executionPlanConfigurationDto.getName(), "testPlan");
+
+            //Remove artifacts
+            eventStreamManagerAdminServiceClient.removeEventStream("org.wso2.sample.pizza.order", "1.0.0");
+            Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(), --eventStreamCount);
+            eventStreamManagerAdminServiceClient.removeEventStream("outStreamNew", "1.0.0");
+            Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(), --eventStreamCount);
+            eventProcessorAdminServiceClient.removeInactiveExecutionPlan("testPlan.siddhiql");
+            Assert.assertEquals(eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount(), --executionPlanCount);
+
+        } catch (Throwable e) {
+            log.error("Exception thrown: " + e.getMessage(), e);
+            Assert.fail("Exception: " + e.getMessage());
+        } finally {
+            eventStreamCount = 0;
+            executionPlanCount = 0;
+        }
+    }
+
+    @Test(groups = {"wso2.cep"}, description = "Testing get siddhi streams and set statistics/tracing enable")
+    public void testGetSiddhiStreamsAndSetStatisticsAndTracingEnable() throws Exception {
+        String samplePath = "admin" + File.separator + "EventProcessorAdminServiceTestCase";
+
+        try {
+            eventStreamCount = eventStreamManagerAdminServiceClient.getEventStreamCount();
+            executionPlanCount = eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount();
+
+            //Add input stream definition
+            String pizzaStreamDefinition = getJSONArtifactConfiguration(samplePath, "org.wso2.sample.pizza.order_1.0.0.json");
+            eventStreamManagerAdminServiceClient.addEventStreamAsString(pizzaStreamDefinition);
+            Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(), ++eventStreamCount);
+
+            //Add output stream definition
+            String outStreamDefinition = getJSONArtifactConfiguration(samplePath, "outStream_1.0.0.json");
+            eventStreamManagerAdminServiceClient.addEventStreamAsString(outStreamDefinition);
+            Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(), ++eventStreamCount);
+
+            //Add execution plan
+            String executionPlan = getExecutionPlanFromFile(samplePath, "testPlan.siddhiql");
+            eventProcessorAdminServiceClient.addExecutionPlan(executionPlan);
+            Assert.assertEquals(eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount(), ++executionPlanCount);
+
+            //Set statistics and tracing enable
+            eventProcessorAdminServiceClient.setStatisticsEnabled("testPlan", true);
+            eventProcessorAdminServiceClient.setTracingEnabled("testPlan", true);
```

Comment:
@thiliA 
Would it be possible to verify that these flags have been set by retrieving and verifying in ExecutionPlanConfigurationDto?